web/src/ProviderEmbeddingTest.js

```diff
@@ -0,0 +1,207 @@
+import React from "react";
+import {App, Button, Card, Divider, Input, message} from "antd";
+import {Typography} from "antd";
+import i18next from "i18next";
+import * as Setting from "./Setting";
+import * as VectorBackend from "./backend/VectorBackend";
+
+const {TextArea} = Input;
+const {Text} = Typography;
+
+class ProviderEmbeddingTest extends React.Component {
```

Comment:
Reuse this: https://ai-admin.casibase.com/vectors/vector_wokw8n

<img width="3734" height="1890" alt="image" src="https://github.com/user-attachments/assets/9c254e29-5c47-49fb-9740-940f10e0e7d2" />